### PR TITLE
misc: Adapt docs and use correct reserve amount

### PR DESCRIPTION
**Pull request by mustermeiszer**
Just fixing using `pool.reserve.total` instead of `pool.reserve.available`.

Minors: 
* added docs for the future
* removed redundant calculation between pannel and card

### CODE REVIEW

File: centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx [hunk 1]
```diff
@@ -195,6 +195,11 @@ export function NavManagementAssetTable({ poolId }: { poolId: string }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [initialValues, isEditing, isLoading])
 
+
+  // NOTE: This assumes that pool.reserve.total comes from onchain state AND NOT from the runtime-apis
```
Comment (mustermeiszer) [inline centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx:199]: @onnovisser or @sophialittlejohn is this correct? Does the `pool.reserve.total` come from onchain state? That is critical here.

Comment (sophialittlejohn) [reply centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx:199]: the value you're using definitely comes from onchain 
https://github.com/centrifuge/apps/blob/15ac6108c79468ce668f1f1ff983f53bef10595b/centrifuge-js/src/modules/pools.ts#L2092

Comment (mustermeiszer) [reply centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx:199]: Perfect. As long as it is not coming from `api.PoolsApi.nav()` we are good.

**Comment by mustermeiszer** [comment]
Can we merge this then?

**Review by onnovisser — APPROVED** [review]
LGTM

**Pull request merged**